sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/instrumentation/LogAndDelegatePersister.java

```diff
@@ -0,0 +1,77 @@
+package com.mesosphere.sdk.scheduler.instrumentation;
+
+import com.mesosphere.sdk.storage.Persister;
+import com.mesosphere.sdk.storage.PersisterException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collection;
+import java.util.Map;
+
+/**
+ * Logs each operation and delegates to the actual persister.
+ */
+public class LogAndDelegatePersister implements Persister {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LogAndDelegatePersister.class);
+    private final Persister targetPersister;
+
+    public LogAndDelegatePersister(Persister targetPersister) {
+        this.targetPersister = targetPersister;
+    }
+
+    private void log(String methodName) {
```

Comment:
These util methods could be static (both here and in other classes below).

If you wanted, could even break them out into a separate utility class. I'd imagine at some point you'd want to start tracking how long calls were taking to complete, etc. The common timer functionality for this could be provided in the utility class.